modules/network/src/main/java/org/apache/ignite/internal/network/direct/stream/DirectByteBufferStreamImplV1.java

```diff
@@ -1343,6 +1362,81 @@ public IgniteUuid readIgniteUuid() {
         return map0;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public <T> void writeMarshallable(T object, MessageWriter writer) {
+        switch (marshallableState) {
+            case 0:
```

Comment:
It would look alien in this class

Reply:
and, well, why do they need clarifying names?

Reply:
It looks like these are stage numbers which are local to each method using them. For example, in this case the stages are 'writing missing descriptors' and 'writing marshalled object'. I still think that it would be nice to explicitly name them and use something like `nextStage()` instead of just `++`, and I think that this should be applied to those the class. But this is probably too much in the context of this task, so probably such a refactoring might be postponed (if needed at all).